src/foraging_gui/Foraging.py

```diff
@@ -1389,17 +1388,13 @@ def _InitializeBonsai(self):
                 # We could connect
                 logging.info('Connected to Bonsai after {} seconds'.format(wait))
                 logging.info('Bonsai started successfully')
-                if self.WarningLabel.text() == 'Lost bonsai connection':
-                    self.WarningLabel.setText('')
-                    self.WarningLabel.setStyleSheet(self.default_warning_color)
                 self.InitializeBonsaiSuccessfully=1
                 subprocess.Popen('title Box{}'.format(self.box_letter),shell=True)
                 return
 
         # Could not connect and we timed out
         logging.info('Could not connect to bonsai with max wait time {} seconds'.format(max_wait))
-        self.WarningLabel_2.setText('Started without bonsai connected!')
-        self.WarningLabel_2.setStyleSheet(self.default_warning_color)
```

Comment:
![image](https://github.com/user-attachments/assets/ffdf748f-8830-4730-802f-85b3cfeaee00)

Reply:
This photo of WarningLabel_2 should sufficiently depict what WarningLabel_2 looks likes, even though different text is assigned to it.